config/tools-manual.json

```diff
@@ -81,12 +81,12 @@
                 "title": "Springwolf",
                 "description": "Automated documentation for async APIs built with Spring Boot. Like Springfox for AsyncAPI. Auto-generates an AsyncAPI document and a web UI.",
                 "links": {
-                    "websiteUrl": "https://springwolf.github.io/",
+                    "websiteUrl": "https://www.springwolf.dev",
```

Comment:
please also update the address for another entry in this document. Springwolf is listed under 2 different categories